pkg/version/postgrest.go

```diff
@@ -22,7 +22,9 @@ type SwaggerResponse struct {
 	Info    SwaggerInfo `json:"info"`
 }
 
-func (t *TenantAPI) GetPostgrestVersion(ctx context.Context) (string, error) {
+func (t *ServiceGateway) GetPostgrestVersion(ctx context.Context) (string, error) {
+	// Cannot use admin endpoint because cloudflare overwrites server response header
+	// Ref: https://postgrest.org/en/v11/references/admin.html#server-version
```

Comment:
**praise:**
That's the kind of comments that are helpful when you dig into those things with little context 👍